src/main/java/org/bihealth/mi/easybus/implementations/email/ConnectionEmail.java

```diff
@@ -388,7 +388,7 @@ protected String getSendingUserName() {
      * @return
      * @throws InterruptedException 
      */
-    protected List<BusMessage> receive(MessageFilter filter) throws BusException, InterruptedException {
+    protected synchronized List<BusMessage> receive(MessageFilter filter) throws BusException, InterruptedException {
```

Comment:
Does this really need to be synchronized? Why? If, yes: please describe the reason in the method comment.